content/guides/identity-management/quick-start/2-configure-okta.md

```diff
@@ -0,0 +1,262 @@
+---
+type: quick-start
+hide_in_page_nav: true
+---
+
+# Configure Okta
+
+Our next step in the Okta / Box integration is to create and configure the Okta
+application and users, then extract a few pieces of information that we will
+need to connect to Okta in our application.
+
+<ImageFrame noborder center shadow>
+  ![Okta Application Dashboard](./img/okta-applications.png)
+</ImageFrame>
+
+For this tutorial we will be starting with a blank Okta application and user
+dashboard to avoid any negative effects on existing installations that may be
+in place, and to ensure that we have admin rights to the instance.
+
+## Create an Okta Application
+
+Starting from the [Okta developer site][okta-dev], sign up for a new developer
+account, or log in under your personal account if you already have one.
+
+After logging in, you should see the Okta dashboard. Click on the **Admin**
+button at the top right.
+
+<ImageFrame noborder center shadow>
+  ![Okta Application Dashboard](./img/okta-qs-step2-dashboard.png)
+</ImageFrame>
+
+You should now see the admin panel. Click on the **Applications** option at the
+top.
+
+<ImageFrame noborder center shadow>
+  ![Okta Admin Dashboard](./img/okta-qs-step2-admin-dashboard.png)
+</ImageFrame>
+
+On the application page, click the **Add Application** button. Select **Web**
+as the application type and click the **Next** button.
+
+<ImageFrame noborder center shadow>
+  ![Okta App Types](./img/okta-qs-step2-app-type.png)
+</ImageFrame>
+
+Okta employs both [OAuth 2][oauth2] and [OpenID Connect][openid-connect] (OIDC)
+for application authorization and user authentication, respectively. The OpenID
+Connect integration allows us to use baked in OIDC connectors within a number
+of popular language frameworks to simplify application and user management by
+handling the callback routes and providing methods for logging in,
+logging out, and protecting routes into your application.
+
+To simplify this first integration, we're going to use the default callback
+routes and settings for the language and framework OIDC connector. Depending on
+your preferred integration type, the configuration settings will slightly
+change.
+
+Fill out the application details with the following configuration settings:
+
+<Choice option='programming.platform' value='node' color='none'>
+
+* Name: Any
+* Base URIs: `http://localhost:3000/`
+* Login redirect URIs: `http://localhost:3000/authorization-code/callback`
+* Logout redirect URIs: `http://localhost:3000/logout`
+* Grant type allowed: Only **Authorization Code** selected
+
+<ImageFrame noborder center shadow>
+  ![Okta App Configuration](./img/okta-qs-step2-appconfig-node.png)
```

Comment:
This image doesn't seem to match the grant type you suggested above. The image shows all tick boxes selected.

Reply:
Fixed